From e36f79439be76f125b67081eb4605b377d738cca Mon Sep 17 00:00:00 2001
Message-Id: <e36f79439be76f125b67081eb4605b377d738cca.1505389490.git.jan.steffens@gmail.com>
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Thu, 14 Sep 2017 07:36:37 +0200
Subject: [PATCH] meson: Fix permissions of installed scripts

configure_file preserves the attributes, so the templates need to be
executable for the (installed) outputs to be executable.
---
 gio/gdbus-2.0/codegen/gdbus-codegen.in | 0
 gio/gdbus-2.0/codegen/meson.build      | 1 -
 glib-gettextize.in                     | 0
 gobject/meson.build                    | 1 -
 4 files changed, 2 deletions(-)
 mode change 100644 => 100755 gio/gdbus-2.0/codegen/gdbus-codegen.in
 mode change 100644 => 100755 glib-gettextize.in

diff --git a/gio/gdbus-2.0/codegen/gdbus-codegen.in b/gio/gdbus-2.0/codegen/gdbus-codegen.in
old mode 100644
new mode 100755
diff --git a/gio/gdbus-2.0/codegen/meson.build b/gio/gdbus-2.0/codegen/meson.build
index 0e9ffbdfade70779..93d9106adba17a89 100644
--- a/gio/gdbus-2.0/codegen/meson.build
+++ b/gio/gdbus-2.0/codegen/meson.build
@@ -13,7 +13,6 @@ gdbus_codegen_conf.set('VERSION', glib_version)
 gdbus_codegen_conf.set('PYTHON', python.path())
 
 # Install gdbus-codegen executable
-# FIXME: Set permissions
 gdbus_codegen = configure_file(input : 'gdbus-codegen.in',
   output : 'gdbus-codegen',
   install : true,
diff --git a/glib-gettextize.in b/glib-gettextize.in
old mode 100644
new mode 100755
diff --git a/gobject/meson.build b/gobject/meson.build
index 0d39f2bbc36f816e..01f37a33a623f4b8 100644
--- a/gobject/meson.build
+++ b/gobject/meson.build
@@ -76,7 +76,6 @@ python_tools_conf.set('VERSION', glib_version)
 python_tools_conf.set('PYTHON', python.path())
 
 foreach tool: python_tools
-  # FIXME: Ensure we set the appropriate permissions
   tool_bin = configure_file(
     input : tool + '.in',
     output : tool,
-- 
2.14.1

